web_src/js/features/repo-projects.js

```diff
@@ -6,57 +6,54 @@ async function initRepoProjectSortable() {
 
   const {Sortable} = await import(/* webpackChunkName: "sortable" */'sortablejs');
   const boardColumns = document.getElementsByClassName('board-column');
-  new Sortable(
-    els[0],
-    {
-      group: 'board-column',
-      draggable: '.board-column',
-      animation: 150,
-      ghostClass: 'card-ghost',
-      onSort: () => {
-        const board = document.getElementsByClassName('board')[0];
-        const boardColumns = board.getElementsByClassName('board-column');
-
-        boardColumns.forEach((column, i) => {
-          if (parseInt($(column).data('sorting')) !== i) {
-            $.ajax({
-              url: $(column).data('url'),
-              data: JSON.stringify({sorting: i, color: rgbToHex($(column).css('backgroundColor'))}),
-              headers: {
-                'X-Csrf-Token': csrfToken,
-                'X-Remote': true,
-              },
-              contentType: 'application/json',
-              method: 'PUT',
-            });
-          }
-        });
-      },
-    },
-  );
 
-  for (const column of boardColumns) {
-    new Sortable(
-      column.getElementsByClassName('board')[0],
-      {
-        group: 'shared',
-        animation: 150,
-        ghostClass: 'card-ghost',
-        onAdd: (e) => {
-          $.ajax(`${e.to.dataset.url}/${e.item.dataset.issue}`, {
+  new Sortable(els[0], {
+    group: 'board-column',
+    draggable: '.board-column',
+    animation: 150,
+    ghostClass: 'card-ghost',
+    onSort: () => {
+      const board = document.getElementsByClassName('board')[0];
+      const boardColumns = board.getElementsByClassName('board-column');
+
+      for (const [i, column] of boardColumns.entries()) {
```

Comment:
Here introduces a bug: `boardColumns` is `HTMLCollectionOf<Element>`, https://stackoverflow.com/questions/22754315/for-loop-for-htmlcollection-elements

I proposed a fix (and the PR fixes more):
* https://github.com/go-gitea/gitea/pull/17753

Reply:
Ah, you're right `HTMLCollection` does not implement `.entries`. One way of fixing it could be `Array.from(boardColumns).entries()`.

Reply:
I just wrote `for (i = 0; i < len; i++)`  😂

Reply:
Fine too I guess, probably better performance with a classic `for` loop.